book/src/contributing/rust_opaque_type_safety.md

```diff
@@ -1,9 +1,9 @@
-# Opaque type safety
```

Comment:
btw, would be great to add a desing doc about DartOpaque since this is a great (thus hard-to-understand for new learners) PR!